jekyll/_cci2/insights.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: classic-docs
+title: "Using Insights"
+short-title: "Using Insights"
+description: "Viewing the status of repos and test performance"
+categories: [configuring-jobs]
+order: 41
+---
+
+*[Basics]({{ site.baseurl }}/2.0/basics/) > Using Insights*
+
+This document describes creating and using insights in CircleCI in the following sections:
+
+* TOC
+{:toc}
+
+## Overview
+
+CircleCI Insights shows you how your tests are performing with integrated graphs and a dashboard for performance of all of your code repositories.
+
+![header]({{ site.baseurl }}/assets/img/docs/insights-1.0.gif)
+
+The image illustrates the following data about your builds:
+
+- Status of all your repos building on CircleCI in real time
+
+- Median queue time
+
+- Median build time
+
+- Number of branches
+
+- Last build
+
+## Project Insights
+
+Click the Insights icon on the main navigation, then click your repo name to access per-project insights.
+
+The per-project insights page gives you access to the build status and build performance graphs for a selected branch.
+
+![header]({{ site.baseurl }}/assets/img/docs/insights-current-build.png)
+
+- **Build Status:** The Insights dashboard shows the last 50 builds for your default branch. Click a branch to access over 100 build statuses for the selected branch.
```

Comment:
Click a branch on the top right corner to access over 100 build/job statuses for the selected branch.

Reply:
thanks, added